adam/experiment/__init__.py

```diff
@@ -143,6 +169,19 @@ def execute_experiment(
         logging.warning(f"Starting point {starting_point} is invalid, setting to 0")
         starting_point = 0
 
+    if resume_from_latest_logged_state and load_learner_state is not None:
+        raise RuntimeError(
+            "Cannot both resume from latest logged state and load a specified state file."
+        )
+
+    if resume_from_latest_logged_state and learner_logging_path is None:
+        raise RuntimeError(
+            "Need state logging path to be able to resume from latest learner state."
+        )
+
+    if resume_from_latest_logged_state and starting_point != -1:
+        raise RuntimeError("Starting point should not be specified when")
```

Comment:
This error message looks incomplete.

Reply:
And it is. :)

Thanks for catching this.

Reply:
This should be fixed now.